pkg/pillar/zedcloud/send.go

```diff
@@ -133,16 +134,28 @@ func SendOnAllIntf(ctx *ZedCloudContext, url string, reqlen int64, b *bytes.Buff
 // Otherwise we test non-free interfaces also.
 // We return a bool remoteTemporaryFailure for the cases when we reached
 // the controller but it is overloaded, or has certificate issues.
+// Return Values:
+//    success/Failure, remoteTemporaryFailure, error, intfErrMap
+//      If Failure,
+//          remoteTemporaryFailure indicates if it is a temp. one
+//          error indicates details of Errors
+//      map[string]types.ErrorInfo - map[string]types.ErrorInfo
+//          - This has per-interface errors
 func VerifyAllIntf(ctx *ZedCloudContext,
-	url string, successCount int, iteration int) (bool, bool, error) {
+	url string, successCount int,
+	iteration int) (bool, bool, types.PerInterfaceErrorMap, error) {
 	var intfSuccessCount int = 0
 	const allowProxy = true
 	var errorList []error
+
 	remoteTemporaryFailure := false
+	// Map of per-interface errors
+	intfErrMap := *types.NewPerInterfaceErrorMap()
+	errSrc := "Network Test All Interfaces"
```

Comment:
That isn't the meaning of ErrorSource in the rest of pillar. Please delete.

Reply:
What is the right meaning? Isn;t errorSource supposed to. indicate which component we got the error from?

Reply:
We talked about this on slack.